.github/workflows/run_tests.yaml

```diff
@@ -35,3 +35,17 @@ jobs:
       run: python -m pip install tox
     - name: Run tests
       run: tox -e unit
+  integration-test-microk8s:
+    name: Integration tests (microk8s)
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup operator environment
+      uses: charmed-kubernetes/actions-operator@main
+      with:
+        provider: microk8s
+    - name: Run Grafana-agent tests
+      run: tox -e integration
+    - name: Run lma bundle tests
+      run: tox -e integration-lma
```

Comment:
Is this charm (grafana-agent-operator) going to have any impact on the lma bundle tests? i.e. why run `integration-lma` here?

Reply:
AFAIU yes, but will remove it now, and add it later when we write the tests for that